### PR TITLE
chore: automated localization updates

### LINKED ISSUE #7821: Chore: update l10n files to pick up more strings
The latest release has a large number of new strings; they're not all going to be translated in time for R13, but this issue is here to match up with the existing PR that handles more of the strings that are going in.

### PR DESCRIPTION
**Pull request by GeoffCoxMSFT**
##Description This is the localization ADO build pipeline pull request.

closes #7821

### CODE REVIEW

**Comment by beyackle** [comment]
Hang on, this one might actually have worked.

**Comment by beyackle** [comment]
No, never mind. The files are correct in ADO but the PR isn't picking them up.

File: Composer/packages/server/src/locales/en-US.json [hunk 1]
```diff
@@ -1703,8 +1709,8 @@
   "for_properties_of_type_list_or_enum_your_bot_accep_9e7649c6": {
     "message": "For properties of type list (or enum), your bot accepts only the values you define. After your dialog is generated, you can provide synonyms for each value."
   },
-  "for_security_purposes_your_bot_can_only_call_a_ski_4b0c81e0": {
-    "message": "For security purposes your bot can only call a skill if it’s Microsoft App Id is in apps allowed callers list. Once you create a publishing profile share your bot’s App ID with the skill’s author to add it to the skill’s allowed callers list. You may also need to include the skill’s app Id in the root bot’s allowed callers list."
+  "for_security_purposes_your_bot_can_only_call_a_ski_a600c3ce": {
+    "message": "For security purposes, your bot can only call a skill if its Microsoft App ID is in the skill''s allowed callers list. Once you create a publishing profilem share your bot’s App ID with the skill’s author to add it. You may also need to include the skill’s App Id in the root bot’s allowed callers list."
```
Comment (GeoffCoxMSFT) [inline Composer/packages/server/src/locales/en-US.json:1713]: Does double quote do the curly quote here?  I've been putting in the curly quote character.

**Review by cwhitten — APPROVED** [review]

**Comment by cwhitten** [comment]
Approved with the understanding that this has been spot-checked inside Electron with languages that have been giving us grief i.e. French, German, Korean, etc.

**Comment by beyackle** [comment]
> Approved with the understanding that this has been spot-checked inside Electron with languages that have been giving us grief i.e. French, German, Korean, etc.

It has, with none of the usual problems that plagued us.

**Pull request merged**